src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -2,52 +2,100 @@ import { useEffect } from 'react';
 
 import type { Theme } from '@mui/material';
 import Grid from '@mui/material/Grid';
-import { Container } from '@mui/system';
+import { useLocation } from 'react-router-dom';
 import { useSearchParams } from 'react-router-dom';
+import { style } from 'typestyle';
 
+import { compareView } from '../../../common/constants';
 import { useAppDispatch } from '../../../hooks/app';
+import useFetchCompareResults from '../../../hooks/useFetchCompareResults';
+import useHandleChangeSearch from '../../../hooks/useHandleChangeSearch';
 import { switchToFakeData } from '../../../reducers/CompareResults';
+import { clearSelectedRevisions } from '../../../reducers/SelectedRevisionsSlice';
+import { SearchContainerStyles } from '../../../styles';
+import { background } from '../../../styles';
+import { Repository } from '../../../types/state';
+import CompareWithBase from '../../Search/CompareWithBase';
 import PerfCompareHeader from '../../Shared/PerfCompareHeader';
-import InputsReplacement from './InputsReplacement';
 import ResultsMain from './ResultsMain';
 
-
+interface ResultsViewProps {
+  protocolTheme: Theme;
+  toggleColorMode: () => void;
+}
 function ResultsView(props: ResultsViewProps) {
+  const dispatch = useAppDispatch();
   const { protocolTheme, toggleColorMode } = props;
   const themeMode = protocolTheme.palette.mode;
+  const view = compareView;
+  const styles = {
+    container: style({
+      backgroundColor: background(themeMode),
+    }),
+  };
   const [searchParams] = useSearchParams();
   const fakeDataParam: string | null = searchParams.get('fakedata');
-  const dispatch = useAppDispatch();
+
 
 // TODO: Populate store with real data or fake data pased on URL params
   useEffect(() => {
     if (fakeDataParam === 'true') {
         dispatch(switchToFakeData());
     }
   }, [fakeDataParam]);
-  
+
+  const sectionStyles = SearchContainerStyles(themeMode, view);
+
+  const location = useLocation();
+  const { dispatchFetchCompareResults } = useFetchCompareResults();
+  const { searchByRevisionOrEmail } = useHandleChangeSearch();
+
+  useEffect(() => {
+    const urlSearchParams = new URLSearchParams(location.search);
+    const repos = urlSearchParams.get('repos')?.split(',');
+    const revs = urlSearchParams.get('revs')?.split(',');
+
+    if (revs && repos) {
+      dispatch(clearSelectedRevisions());
```

Comment:
I kind of wonder if we want to clear them. What if the user wants to move back to the previous page, do we want them to select them again, or rather that they see the same revisions already selected? What are your thoughts on that?

Reply:
Actually now that I'm looking at it, this isn't necessary at all. Whether or not the user moves back to the previous page, the home page, they still won't see the same revisions. They have to check the revisions they want again.